### DISCUSSION TITLE
Write newsletter article about recycling clothes

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#1110] [Forum]
This is a todo from #799 

It was an idea by @tangerineshark with support from @DocSpencer77 so I'm assigning this to them.

**Comment by tangerineshark** [comment]
I think this might have been more @rrolon47 than me, although I'm happy to help!

**Comment by DocSpencer77** [comment]
@tangerineshark @joostdecock  Karen, are you willing to talk about and test a freesewing version of my "[Chullo Hat](https://www.aracari.com/blog/luxury-travel-peru/the-iconic-andean-chullo-hats/#:~:text=The%20chullo%20is%20the%20iconic,elements%20of%20the%20mountainous%20region.)"?  I made my last few from polar fleece but have been thinking it would be great to recycle wool sweaters by cutting them apart and turning them into "boiled wool", then cutting them out with a pattern.  Is your head measurement ~22"?  <g>  You're the only person to show interest in testing a hat pattern, so I'm not going to forget that.

**Comment by tangerineshark** [comment]
Sure thing @DocSpencer77 ! And yep, my head measurement is 22". 😄

**Comment by joostdecock** [comment]
> I think this might have been more @rrolon47 than me, although I'm happy to help!

I'm sorry, I didn't take sufficient notes during the call :see_no_evil:

**Comment by DocSpencer77** [comment]
@joostdecock  We'll consider it as a good sign you were enjoying the conversation.  😄 

@tangerineshark Have you boiled sweaters before to make boiled wool?  My sister did some of that about a year ago with great results.  I have an old dark brown sweater that I'm willing to sacrifice, and I'm curious about trying to sew a chullo hat with wool felt but wonder if it's flexible enough.

This is the sort of thing I'm considering testing.  On my last two, I used the same weight polar fleece for both the outer fabric and the lining, and while it makes the hats very warm, the extra bulk makes it reasonable to cut the lining/interior just a tiny bit smaller than the exterior. Any ideas for a different lining? I do have some lighter weight fleece.  Maybe I'll try that.

**Comment by rrolon47** [comment]
@DocSpencer77  and are are already talking about ideas for the article.  I will note that I have a few deadlines in the coming weeks so this is not high priority atm.

**Comment by DocSpencer77** [comment]
@rrolon47  I have a couple of extra huge "Indian Motorcycle" t-shirts that I nominate for photogenic recycling. Surely I can find them and take a picture,  Surely. They're not really "old" in the sense of having been worn... just in the sense they've been waiting to be re-fitted for ~4 years.

**Comment by ttimearl** [comment]
I'm happy to help with editing, organisation, style etc. Don't worry about polishing or trying to make it look perfect. If you get to a point where you need a second eye on something, let me know.  

On the newsletter in general, we should probably think about things like: layout, article length, regular sections, etc. These could then generate some contributor guidelines. I know it's early days, but I think starting off with a few basic ideas will help in the long run.

**Comment by DocSpencer77** [comment]
@rrolon47 @tangerineshark  In our last call, I mentioned one of my "restoration" projects (primarily to repair or "redo" or embellish knitted/woolen things) involved needle felting.  My first experiment was an old Merino wool sock.  How interesting it was. You can see where the more orange wool roving is integrated into the heel and toe.  It reinforces the area but is smooth and comfortable.  Works wonderfully!  
![FeltingSockRepair2021-02-14 161019](https://user-images.githubusercontent.com/43393580/107890566-4d629200-6edf-11eb-8821-27cd6b21a526.jpg)

You can see the difference in color on the sock, but on my next project--a gray cashmere sweater--the repair is close to invisible.  Now I'd be glad to pick up a slightly damaged sweater knowing I could easily restore it.  And soon I'll have several "renewed" socks!

**Comment by DocSpencer77** [comment]
PS When I comment here, does @joostdecock get a copy even if I don't @ mention him?
PPS I did get the mysterious powerdered "melt it together powder" stuff, but haven't used it. So far, the felting has wrought fine results.  But the results with some other fibers may not be as good as Merino Wool and Cashmere.  Experimentation R Us.  I'll find a use case and hope any mending purists hearabout are not scandalized.  😉

**Comment by joostdecock** [comment]
He does, but he gets dozen of github notifications every day, so manage your expectations :joy:

**Comment by starfetch** [comment]
This seems like an issue that could be better placed in discussions, so I'm moving it over there. Feel free to reopen as an issue if you feel that's more appropriate :)

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
